account_edit_modal.php

```diff
@@ -21,6 +21,25 @@
                         </div>
                     </div>
 
+                    <div class="form-group">
+                        <label>Account Type <strong class="text-danger">*</strong></label>
+                        <div class="input-group">
+                            <div class="input-group-prepend">
+                                <span class="input-group-text"><i class="fa fa-fw fa-list"></i></span>
+                            </div>
+                            <select class="form-control select" name="type" required>
```

Comment:
Something is broken here, the edit modal isn't pre-selecting the current account type...

![image](https://github.com/itflow-org/itflow/assets/32306651/4df40304-ecbf-4cdf-abe5-9291e417f28a)

![image](https://github.com/itflow-org/itflow/assets/32306651/1324a2e8-9611-4e07-afcf-98da258bff57)

Reply:
So I need to restructure how the dropdown is formatted, I think I am additionally going to adjust the account types being hard coded, rather a table in the DB, with a settings page to edit specific accounts, key values, and types. Working on this in the coming days. You can make this PR a draft again.

Reply:
Above work done.